```
   1) Frontend behavior tests
       extended routes.yaml (5): rss override
         serve /podcast/rss/:

      AssertionError: expected 'application/rss+xml' to equal 'application/rss+xml; charset=utf-8'
      + expected - actual

      -application/rss+xml
      +application/rss+xml; charset=utf-8
      
      at Assertion.fail (/home/runner/work/Ghost/Ghost/node_modules/should/cjs/should.js:275:17)
      at Assertion.value (/home/runner/work/Ghost/Ghost/node_modules/should/cjs/should.js:356:19)
      at /home/runner/work/Ghost/Ghost/ghost/core/test/regression/mock-express-style/api-vs-frontend.test.js:1563:61
```

Interesting. I have never changed anything regarding the `charset=utf-8` part.
Should I just update the test to check for `application/rss+xml`, or how should this be fixed?

This would be the "easiest" way:
```diff
# File ghost/core/test/regression/mock-express-style/api-vs-frontend.test.js
# Line 1559

            return localUtils.mockExpress.invoke(app, req)
                .then(function (response) {
                    response.statusCode.should.eql(200);
                    response.template.should.eql('podcast/rss');
-                    response.headers['content-type'].should.eql('application/rss+xml; charset=utf-8');
+                    response.headers['content-type'].should.eql('application/rss+xml');
                    response.body.match(/<link>/g).length.should.eql(2);
                });
```

--- a/ghost/core/test/regression/mock-express-style/api-vs-frontend.test.js
+++ b/ghost/core/test/regression/mock-express-style/api-vs-frontend.test.js
@@ -1560,7 +1560,7 @@ describe('Frontend behavior tests', function () {
                 .then(function (response) {
                     response.statusCode.should.eql(200);
                     response.template.should.eql('podcast/rss');
-                    response.headers['content-type'].should.eql('application/rss+xml; charset=utf-8');
+                    response.headers['content-type'].should.eql('application/rss+xml');
                     response.body.match(/<link>/g).length.should.eql(2);
                 });
         });

